src/components/entity-panel/sections/tactic/TheTacticListContent.vue

```diff
@@ -2,55 +2,85 @@
   <span v-if="!items.length" class="caption px-2" v-text="$t('tactic.noTacticsFound')"/>
   <v-treeview
     v-else
-    v-model="activeElements"
+    :active="[currentTacticId]"
     :items="items"
-    :open="open"
     class="pa-0"
     dense
     open-on-click
     activatable
+    @update:active="switchTactic($event)"
   >
     <template v-slot:prepend="{ item, isOpen }">
       <v-icon v-if="item.children">
         {{ isOpen ? 'fa-folder-open' : 'fa-folder' }}
       </v-icon>
-      <v-avatar
-        v-else
-        size="30"
-        class="ml-0 pl-0"
-      >
-        <img :src="item.tactic.map.icon">
-      </v-avatar>
+        <v-badge
+          v-else
+          icon="fa-bookmark"
+          :value="item.tactic.isPinned"
```

Comment:
bind under v-else

Reply:
bind under v-else is correct. However bind under non bind isn't.

I have fixed